src/components/tabs/tabs.test.js

```diff
@@ -23,6 +19,13 @@ describe('EuiTabs', () => {
       });
     });
 
+    describe('display', () => {
+      test('is rendered', () => {
```

Comment:
```suggestion
      test('can be condensed', () => {
```

This way the test is more explicit and scalable if we ever add more choices to 'display'.

Reply:
👍